tests/Jobs/PollForMediaWikiJobsJobTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Tests\Jobs;
+
+use Tests\TestCase;
+use App\Wiki;
+use Illuminate\Contracts\Queue\Job;
+use App\Jobs\PollForMediaWikiJobsJob;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Http;
+
+class PollForMediaWikiJobsJobTest extends TestCase
+{
+
+    use RefreshDatabase;
```

Comment:
This (stub) test case passes, but makes subsequent tests fail with odd error messages. This makes me think I introduce some statefulness and still need to clean something, but tbh I don't really know what. Any pointers?